rental.rb

```diff
@@ -1,8 +1,11 @@
+require_relative 'data_related'
+
 # rubocop: disable Layout/LineLength
 # rubocop: disable Style
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `rubocop`, it will be nice to fix the issue.